src/pudl/workspace/setup.py

```diff
@@ -229,7 +227,7 @@ def init(pudl_in, pudl_out, clobber=False):
     deploy(settings_pkg, settings_dir, ignore_files, clobber=clobber)
 
     # Make output directory:
-    pudl_out = pathlib.Path(pudl_settings["pudl_out"]) / "output"
+    pudl_out = pathlib.Path(pudl_settings["pudl_out"])
```

Comment:
@bendnorman I took out the `/ "output"` because it seemed like we were writing files directly to `$PUDL_OUTPUT/` and the test were looking for files in `$PUDL_OUTPUT/output` - but maybe we should be telling the ETL to write things to `$PUDL_OUTPUT` instead?

Reply:
FWIW this *is* uploading to intake correctly 🤷